xamarin/Xamarin.Forms
# Fix datetimepickerlocalization

### Description of Change ###

The Format property of the DatePicker and TimePicker creates consistent changes of the date and time displays. Also, the DatePicker format is consistent with the iOS default for specific region and language settings.

### Issues Resolved ### 

- fixes #11471 
- fixes #11472 
- fixes #12556

### API Changes ###

 None

### Platforms Affected ### 


- iOS
- Android
- UWP

### Behavioral/Visual Changes ###

The DatePicker and TimePicker displays and pickers might change depending on the format property as well as the regional settings.

### Before/After Screenshots ### 

Before:
![pic1timepicker](https://user-images.githubusercontent.com/66331238/87821322-47b3a980-c835-11ea-81d2-5b1aad997a6a.png)

After:
![pic4timepicker](https://user-images.githubusercontent.com/66331238/87821353-55692f00-c835-11ea-9f11-75dcc25b8005.png)

### Testing Procedure ###

Create DatePicker and TimePickers on Android, iOS, and UWP and play around with different format properties as well as different regional settings to see if the date and time picker displays and picker stay consistent and follow the region's culture default format for date and time respectively

### PR Checklist ###
